app/src/main/java/com/github/sdp/ratemyepfl/model/review/Comment.kt

```diff
@@ -0,0 +1,62 @@
+package com.github.sdp.ratemyepfl.model.review
+
+import com.github.sdp.ratemyepfl.model.time.DateTime
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Comment constructor(
+    val subjectId: String = "",
+    override val comment: String = "",
+    override val date: DateTime = DateTime.DEFAULT_DATE_TIME,
+    override val uid: String? = null,
+    override var likers: List<String> = listOf(),
+    override var dislikers: List<String> = listOf()
+) : Post("", comment, date, uid, likers, dislikers) {
```

Comment:
I guess that it is a Post for UI/adapter usage, but since it does not have a title is it worth the inheritance? I mean does it introduce some overhead to have two distinct classes? Maybe a more general interface above like Postable or whatever would be easier.

Reply:
Agreed but it's a very small detail

Reply:
Yes I agree :)